content/docs/introduction/first_steps.md

```diff
@@ -162,7 +162,13 @@ Now we restart our Prometheus server to activate our new job.
 
 Go to the expression browser and verify that Prometheus now has information
 about the time series that this endpoint exposes. Navigate to
-http://localhost:9090/graph and use the dropdown next to the "Execute" button to see a list of metrics this server is collecting. In the list you'll see a number of metrics prefixed with `node_`, that have been collected by the Node Exporter by our `node` job. For example, you can see the node's CPU usage via the `node_cpu_seconds_total` or `node_exporter_build_info` metric. 
+http://localhost:9090/graph and use the dropdown next to the "Execute" button to see a list of metrics this server is collecting. In the list you'll see a number of metrics prefixed with `node_`, that have been collected by the Node Exporter by our `node` job. For example, `node_exporter_build_info` shows the version information of the exporter.
+
+You can see the node's CPU usage via the `node_cpu_seconds_total` metric. This is a typical counter metric, which can be transformed into a usage graph by applying some functions.
+
+```
+sum(rate(node_cpu_seconds_total{mode!="idle"}[5m])) without (cpu,mode)
```

Comment:
Our style is that without goes with the sum

Reply:
Is this officially documented? I know you prefer this way, but other examples do not use this style.

Reply:
FWIW, I prefer the style Ben used.

Reply:
They do: https://prometheus.io/docs/practices/rules/ and it's what the Prometheus server itself uses